jwst/assign_wcs/util.py

```diff
@@ -567,49 +580,55 @@ def create_grism_bbox(input_model,
     else:
         raise ValueError("create_grism_object works with NIRCAM and NIRISS WFSS exposures only.")
 
+    if reference_files is None:
+        # Get the list of extract_orders and lmin, lmax from wavelength_range.
+        if wavelength_range is None:
+            message = "If reference files are not supplied, ``wavelength_range`` must be provided."
+            raise TypeError(message)
+    else:
+        # Get the list of extract_orders and lmin, lmax from the ``wavelengthrange11 reference file.
+        with WavelengthrangeModel(reference_files['wavelengthrange']) as f:
+            if 'WFSS' not in f.meta.exposure.type:
+                err_text = "Wavelengthrange reference file not for WFSS"
+                log.error(err_text)
+                raise ValueError(err_text)
+            ref_extract_orders = f.extract_orders
+            if extract_orders is None:
+                #ref_extract_orders = extract_orders
+                extract_orders = [x[1] for x in ref_extract_orders if x[0] == filter_name].pop()
+
+            wavelength_range = f.get_wfss_wavelength_range(filter_name, extract_orders)
+
+
+    if not isinstance(mmag_extract, (int, float)):
+        raise TypeError(f"Expected mmag_extract to be a number, got {mmag_extract}")
+    log.info("Extracting objects < abmag = {0}".format(mmag_extract))
+
     # extract the catalog objects
     if input_model.meta.source_catalog is None:
-        err_text = "No source catalog listed in datamodel"
+        err_text = "No source catalog listed in datamodel."
         log.error(err_text)
         raise ValueError(err_text)
 
-    log.info("Getting objects from {}".format(input_model.meta.source_catalog))
+    log.info(f"Getting objects from {input_model.meta.source_catalog}")
+
+    return _create_grism_bbox(input_model, mmag_extract, wfss_extract_half_height, wavelength_range)
+
+
+def _create_grism_bbox(input_model, mmag_extract=99.0,
```

Comment:
Do we want mmag hardwired to 99 or allow a user value to be passed in?

Reply:
The default value is 99, a user can change it.